apps/estuary/lib/estuary/message_handler.ex

```diff
@@ -1,8 +1,33 @@
 defmodule Estuary.MessageHandler do
+  @moduledoc """
+  This module will read an event from the event stream and persist it.
+  """
   use Elsa.Consumer.MessageHandler
   require Logger
 
   def handle_messages(messages) do
+    Enum.each(messages, fn message ->
```

Comment:
Can we possibly split some of this logic out into smaller functions to make it more readable? Maybe pipe the result of `Jason.decode` into a function that handles some of the edge/degenerate cases instead of having to do `else`